torch/nn/modules/instancenorm.py

```diff
@@ -59,6 +62,64 @@ def forward(self, input: Tensor) -> Tensor:
             self.training or not self.track_running_stats, self.momentum, self.eps)
 
 
+class _LazyInstanceNorm(LazyModuleMixin, _InstanceNorm):
```

Comment:
While this looks correct to me, there's duplication between this and `_LazyBatchNorm`. How feasible is it to alternatively implement a `_LazyNormBase` from which both `_LazyBatchNorm` and `_LazyInstanceNorm` derive?

Reply:
Done!